code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -531,7 +531,24 @@ GLOBAL_VAR(bomb_set)
 	if(zap_flags & ZAP_MACHINE_EXPLOSIVE)
 		qdel(src)//like the singulo, tesla deletes it. stops it from exploding over and over
 
-#define NUKERANGE 80
+/// Determine the location of the nuke with respect to the station. Used for,
+/// among other things, calculating win conditions for nukies and choosing which
+/// round-end cinematic to play.
+/obj/machinery/nuclearbomb/proc/get_nuke_site()
+	var/turf/bomb_turf = get_turf(src)
+	if(bomb_turf)
+		if(!is_station_level(bomb_turf.z))
+			return NUKE_SITE_OFF_STATION_ZLEVEL
+
+		var/area/bomb_area = get_area(src)
+		for(var/A in GLOB.the_station_areas)
```

Comment:
Actually, can you do this? I think it will be more consistent
```suggestion
		for(var/A in SSmapping.existing_station_areas)
```

Reply:
Done